### DISCUSSION TITLE
Is there a difference in performance when using Rooks useMutationObserver vs the built in MutationObserver?

### DISCUSSION THREAD

**Discussion by jojov92** [imbhargav5/rooks#1580] [Q&A] [answered]
Basically what the q says, is it better for performance if we use Rooks -> `useMutationObserver` rather than using MutationObserver? https://developer.mozilla.org/en-US/docs/Web/API/MutationObserver 
thank you

**Comment by imbhargav5** [comment, marked as answer]
Hi @jojov92 the `useMutationObserver` internally relies on `MutationObserver`. So the performance is pretty much the same. Good luck!
